components/list/list-item-generic.js

```diff
@@ -0,0 +1,73 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemGeneric extends LitElement {
+
+	static get styles() {
+		return css`
+			:host {
```

Comment:
This will need `position: relative;` so that it forces the `z-index` of the elements to fall to 0 outside of this dom element.

Reply:
Grid might make us not have to do this :D

Reply:
Yeah after playing around with this it appears this isn't necessary. 🎉